src/main/java/frc/robot/commands/ArcadeDriveCommand.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Robot;
+import frc.robot.subsystems.Drivetrain;
+
+public class ArcadeDriveCommand extends Command{
+    
+    Drivetrain driv;
+    public ArcadeDriveCommand(Drivetrain drev){
+        driv = drev;
+        addRequirements(driv);
+    }
+
```

Comment:
Can reuse parameter name driv instead of drev in the constructor. In WPILib command-based programming especially, it's better to keep parameter name consistent for clarity. Indicates that the parameter is being assigned to the same instance variable. Makes code easier to read and maintain.

so can have this.driv = driv;

Reply:
Same thing in FunnyCommand. Also remember to make all of the variables private in that command, too.